dev/sg/msp/sg_msp.go

```diff
@@ -451,6 +457,11 @@ The '-handbook-path' flag can also be used to specify where sourcegraph/handbook
 								return errors.Wrapf(err, "load service %q", s)
 							}
 							serviceSpecs = append(serviceSpecs, svc)
+							collectedAlerts, err := CollectAlertPolicies(svc)
+							opts.AlertPolicies = collectedAlerts
+							if err != nil {
+								return errors.Wrapf(err, "%s: CollectAlertPolicies", s)
+							}
 							doc, err := operationdocs.Render(*svc, opts)
```

Comment:
Since `opts` is reused let's just construct it entirely here at the callsite

```suggestion
							doc, err := operationdocs.Render(*svc, operationdocs.Options{
							ManagedServicesRevision: repoRev,
							GenerateCommand:         strings.Join(os.Args, " "),
							Handbook:                true,
							AlertPolicies: collectedAlerts
						}
)
```

Reply:
opts is also used further down as well to render the index page
https://github.com/sourcegraph/sourcegraph/blob/b64422df6dad5af88b4b4107c4d3637454767272/dev/sg/msp/sg_msp.go#L466

Reply:
circling back to this - I think each one should use separate opts in that case. The risk here is that the loop mutates `opts` each time, making the final state of `opts` difficult to grok